chainstate/transaction_info.go

```diff
@@ -1,22 +1,28 @@
 package chainstate
 
 import (
+	"github.com/bitcoin-sv/go-broadcast-client/broadcast"
 	"github.com/libsv/go-bc"
 )
 
 // TransactionInfo is the universal information about the transaction found from a chain provider
 type TransactionInfo struct {
-	BlockHash     string          `json:"block_hash,omitempty"`    // mAPI, WOC
-	BlockHeight   int64           `json:"block_height"`            // mAPI, WOC
-	Confirmations int64           `json:"confirmations,omitempty"` // mAPI, WOC
-	ID            string          `json:"id"`                      // Transaction ID (Hex)
-	MinerID       string          `json:"miner_id,omitempty"`      // mAPI ONLY - miner_id found
-	Provider      string          `json:"provider,omitempty"`      // Provider is our internal source
-	MerkleProof   *bc.MerkleProof `json:"merkle_proof,omitempty"`  // mAPI 1.5 ONLY. Should be also supported by Arc in future
+	BlockHash     string             `json:"block_hash,omitempty"`    // mAPI, WOC
+	BlockHeight   int64              `json:"block_height"`            // mAPI, WOC
+	Confirmations int64              `json:"confirmations,omitempty"` // mAPI, WOC
```

Comment:
I know that this has not changed in this PR but WOC should not be removed? we are not using it anymore

Reply:
All are used. `BlockHash` `BlockHeight` in `sync_tx_service.go` and `Confirmations` by `checkRequirementMapi` function.

Reply:
Sure, I was talking only about the comment `WOC`. Previously this data was probably gathered by WOC but we removed it some time ago or does it refer to something different?

Reply:
I think You're right. @wregulski if you agree too - remove the `WOC` comment please :)

Reply:
done